walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,249 @@
 
 package com.google.sps;
 
+import com.google.sps.TimeRange;
+import java.util.ArrayList;
 import java.util.Collection;
+import java.util.LinkedList;
+import java.util.List;
 
+/**
+ * This class exposes the {@code query} method, which finds available time slots
+ * for a meeting with a list of mandatory and optional attendees.
+ */
 public final class FindMeetingQuery {
+  /**
+   * This method is used to find all available time slots for a meeting,
+   * provided a list of mandatory and optional attendees.
```

Comment:
nit: add [a blank line](https://google.github.io/styleguide/javaguide.html#s7.1.2-javadoc-paragraphs) (i.e. a line only containing `*` between the summary and the @param sections; here and below.